From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Mon, 6 Feb 2017 05:09:47 -0500
Subject: Avoid trying to use git during the build.

Upstream Makefile is too-clever by half.  We want to know the version
during the debian build, we'll use the upstream version.
---
 src/Makefile | 13 ++++---------
 1 file changed, 4 insertions(+), 9 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index 24b2ba0..0406530 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -19,19 +19,14 @@ MAKEFLAGS += --no-print-directory
 endif
 
 version.h:
-	@export GIT_CEILING_DIRECTORIES="$$(readlink -f ../..)" && \
-	ver="#define WIREGUARD_VERSION \"$$(git describe --dirty 2>/dev/null)\"" && \
+	ver="#define WIREGUARD_VERSION \"$(DEB_VERSION_UPSTREAM)\"" && \
 	[ "$$(cat version.h 2>/dev/null)" != "$$ver" ] && \
-	echo "$$ver" > version.h && \
-	git update-index --assume-unchanged version.h || true
+	echo "$$ver" > version.h || true
 
 dkms.conf:
-	@export GIT_CEILING_DIRECTORIES="$$(readlink -f ../..)" && \
-	ver="$$(git describe --dirty 2>/dev/null)" && \
 	. ./dkms.conf && \
-	[ "$$PACKAGE_VERSION" != "$$ver" ] && \
-	sed -i "s/PACKAGE_VERSION=.*/PACKAGE_VERSION=\"$$ver\"/" dkms.conf && \
-	git update-index --assume-unchanged dkms.conf || true
+	[ "$$PACKAGE_VERSION" != "$(DEB_VERSION_UPSTREAM)" ] && \
+	sed -i "s/PACKAGE_VERSION=.*/PACKAGE_VERSION=\"$(DEB_VERSION_UPSTREAM)\"/" dkms.conf || true
 
 module: version.h
 	@$(MAKE) -C $(KERNELDIR) M=$(PWD) modules
